Formula/salmon.rb

```diff
@@ -17,6 +17,7 @@ class Salmon < Formula
   depends_on "automake" => :build
   depends_on "cmake" => :build
   depends_on "boost"
+  depends_on "bowtie2"
```

Comment:
```suggestion
```
I don't believe `salmon` depends on `bowtie2`, since it uses light-weight alignment. I see no reference to `bowtie2` in the documentation https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fsalmon.readthedocs.io+bowtie2

Reply:
Yeah I thought so too, but the docs said the newer version used it in `-validateMappings` mode. But I need to find the docs for that again - i might have misread.

Reply:
I think I imagined it :(